packages/asc-ui/src/components/FormTitle/FormTitle.test.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react'
+import { render } from '@testing-library/react'
+import FormTitle from './FormTitle'
+
+describe('FormTitle', () => {
+  it('should render', () => {
+    const { container, queryByText } = render(<FormTitle>Foo</FormTitle>)
```

Comment:
Should this not be wrapped in a `ThemeProvider`?

Reply:
yep

Reply:
Looks like this is still missing, can you add it as well?